context/AuthContextProvider.tsx

```diff
@@ -17,17 +18,18 @@ type UserCredentials = {
 
 type AuthContextType = {
   isSignedIn: boolean;
-  setIsSignedIn: (isSignedIn: boolean) => void;
-  loginAccount: (user: UserCredentials) => Promise<void | Error>;
+  setIsSignedIn: (isSignedIn: boolean) => void; // eslint-disable-line no-unused-vars
```

Comment:
✅ Done.  

**NOTE:** I fixed setIsSignedIn type definition.  However, could not fix loginAccount due to the variable, user, is needed in type definition since the function is being provided as part of the AuthContext Provider.